### PR TITLE
Dynamic routing with decoupled, centralized data

### PR DESCRIPTION
**Pull request by S3B4S**
Extension of #61

### CODE REVIEW

File: components/Sidebar.vue [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-      nuxt-link(to="/javascript") Javascript
-      nuxt-link(to="/php") PHP
-      nuxt-link(to="/design") Design
-      nuxt-link(to="/frontend") Frontend
-      nuxt-link(to="/fullstack") Fullstack
-      nuxt-link(to="/servers") Server
-      nuxt-link(to="/utility") Utility
-      nuxt-link(to="/daily") Daily
-      nuxt-link(to="/communities") Communities
+      template(v-for='entry in menuEntries')
```
Comment (lostdesign) [inline components/Sidebar.vue]: bit picky here, but imo `category in menuEntries`

File: components/Sidebar.vue [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+        nuxt-link(:to='entry.slug') {{ entry.title }}
 </template>
 
+<script>
+import store from '../store.json'
```
Comment (lostdesign) [inline components/Sidebar.vue]: Just as note, this should be in vuex later. Probs gonna tackle that now or on friday :D

Comment (S3B4S) [reply components/Sidebar.vue]: Agreed. But I do see that as a next step :)

**Comment by lostdesign** [comment]
Why are there merge conflicts now?

**Comment by S3B4S** [comment]
Modified files on master that I have deleted here, I'm resolving these as we speak

File: components/Sidebar.vue (hunk 1 above)
Comment (S3B4S) [reply components/Sidebar.vue]: Agreed, implemented

**Pull request merged**